app/src/main/java/co/develhope/chooseyouowncocktail_g2/ui/home/HomeFragment.kt

```diff
@@ -34,6 +34,8 @@ class HomeFragment : Fragment() {
     private val binding get() = _binding!!
 
     private lateinit var drinkCardAdapter : DrinkCardAdapter
+    private lateinit var concatAdapter : ConcatAdapter
```

Comment:
che cosa è un concatAdapter? forse si potrebbe dare un nome più specifico ed esaustivo

Reply:
dovrebbe essere eliminato in altri brach da altri